pkg/interface/src/views/apps/chat/components/ChatMessage.tsx

```diff
@@ -206,7 +206,9 @@ const MessageWrapper = (props) => {
   return (
     <Box
       py='1'
-      backgroundColor={hovering ? 'washedGray' : 'transparent'}
+      backgroundColor={
+        hovering && props.hoverMarker !== false ? 'washedGray' : 'transparent'
```

Comment:
`false` checks rarely play nice being passed around. Could we rewrite this flipped to check for true?

Reply:
Sure, I'll just invert the flag. No problem